Admin/AbstractAdmin.php

```diff
@@ -422,8 +422,6 @@
      */
     protected $securityInformation = array();
 
-    protected $cacheIsGranted = array();
```

Comment:
@greg0ire the example you showed would not work. Magic methods are only called if property is inaccessible. I can make the property private instead of protected. Moreover, is it a BC break to make the property noop?